src/main.cpp

```diff
@@ -732,59 +753,62 @@ int main ( int argc, char** argv )
            "  -L, --licence         show an agreement window before users can connect\n"
            "  -m, --htmlstatus      enable HTML status file, set file name\n"
-           "  -o, --serverinfo      infos of this server in the format:\n"
+           "  -o, --serverinfo      registration info for this server.  Format:\n"
            "                        [name];[city];[country as QLocale ID]\n"
+           "      --serverpublicip  public IP address for this server.  Needed when\n"
+           "                        registering with a server list hosted\n"
+           "                        behind the same NAT\n"
            "  -P, --delaypan        start with delay panning enabled\n"
            "  -R, --recording       sets directory to contain recorded jams\n"
            "      --norecord        disables recording (when enabled by default by -R)\n"
            "  -s, --server          start server\n"
+           "      --serverbindip    IP address the server will bind to (rather than all)\n"
            "  -T, --multithreading  use multithreading to make better use of\n"
            "                        multi-core CPUs and support more clients\n"
            "  -u, --numchannels     maximum number of channels\n"
-           "  -w, --welcomemessage  welcome message on connect\n"
+           "  -w, --welcomemessage  welcome message to display on connect\n"
+           "                        (string or filename)\n"
            "  -z, --startminimized  start minimizied\n"
-           "      --serverpublicip  specify your public IP address when\n"
-           "                        running a slave and your own directory server\n"
-           "                        behind the same NAT\n"
-           "      --serverbindip    specify the IP address the server will bind to\n"
            "\n"
            "Client only:\n"
+           "  -c, --connect         connect to given server address on startup\n"
+           "  -j, --nojackconnect   disable auto Jack connections\n"
            "  -M, --mutestream      starts the application in muted state\n"
            "      --mutemyown       mute me in my personal mix (headless only)\n"
-           "  -c, --connect         connect to given server address on startup\n"
-           "  -j, --nojackconnect   disable auto JACK connections\n"
+           "      --clientname      client name (window title and jack client name)\n"
            "      --ctrlmidich      MIDI controller channel to listen\n"
-           "      --clientname      client name (window title and JACK client name)\n"
            "\n"
-           "Example: " + QString ( argv[0] ) + " -s --inifile myinifile.ini\n"
+           "Example: %1 -s --inifile myinifile.ini\n"
            "\n"
            "For more information and localized help see:\n"
-           "\n"
-           "https://jamulus.io/wiki/Command-Line-Options\n";
+           "https://jamulus.io/wiki/Command-Line-Options\n"
+    ).arg(argv[0]);
```

Comment:
Styling?

Reply:
?

Reply:
Shouldn’t this be .arg ( argv[0] ); with spaces?

Reply:
Yes, it should - and clang-format should have rejected it.  Hm....

Reply:
Ah, it's the line before the "clang-format on"..!  Oops.